index.html

```diff
@@ -0,0 +1,51 @@
+<html lang="ja">
+  <head>
+    <meta charset="utf-8">
+    <title>1-3</title>
+  </head>
+  <body>
+    
+    <script>
+      'use strict';
+      
+      const items = ['掃除',"買い物","散歩"];
+      const title = ('======================= \n 現在持っているタスク一覧 \n=======================');
+      console.log(title);
+      
+      const itemMap = function(){
+        items.map((value,index) => {
```

Comment:
```suggestion
        items.forEach((value,index) => {
```
`map` ではなくて `forEach` の方が適切と思います。
理由は以下を読んでいただけるとご理解いただけるかと思います。

https://yukimonkey.com/data/map-foreach/